package.json

```diff
@@ -16,7 +16,8 @@
     "lint": "eslint .",
     "prettier": "prettier --write \"**/*.{ts,tsx,cts,mts,js,jsx,cjs,mjs,json}\"",
     "sort-package-json": "sort-package-json package.json \"packages/*/package.json\" \"examples/*/package.json\"",
-    "test": "turbo run test"
+    "test": "turbo run test --filter=!@medplum/docs",
+    "test:docs": "turbo run test --filter=@medplum/docs"
```

Comment:
The docs package.json doesn't have a "test" script, so this should be a no-op, yeah?

Reply:
Good point, I guess I somehow hallucinated it was there at one point, probably because running `npm run test` still triggers a build of `@medplum/docs` currently if the `build` cache gets busted... not sure why. Going to see if `--filter` applies to jobs that get triggered by the current job too

Reply:
Yeah looks like when `build` gets triggered, it still considers `@medplum/docs` within scope and runs `build` for `@medplum/docs` too.

Keeping `--filter=!@medplum/docs` so we don't end up building it when running `npm run test` but can remove `test:docs` of course